docker/docker-compose.yml

```diff
@@ -187,6 +187,44 @@ services:
     networks:
       - mydefault
 
+  rdf-fingerprinter-api:
+    container_name: rdf-fingerprinter-api
+    image: meaningfy/rdf-fingerprinter-api:latest
+    ports:
+      - ${RDF_FINGERPRINTER_API_PORT}:${RDF_FINGERPRINTER_API_PORT}
+    env_file: .env
+    restart: always
```

Comment:
this indicated that if the container "breaks" and quits execution, the `docker-compose` will automatically try to restart it?

Reply:
or if the host system is restarted, then docker containers shall shart on boot